content/en/opentelemetry/otlp_endpoint.md

```diff
@@ -0,0 +1,263 @@
+
+You might prefer this option if you don't want the overhead of using the Datadog Agent or OpenTelemetry Collector.
+
+## Configuration
+
+To export OTLP data to the Datadog OTLP intake endpoint:
+
+1. [Configure the OTLP HTTP Protobuf exporter](#configure-the-exporter).
+   - Set the Datadog OTLP intake endpoint.
+   - Configure the required HTTP headers.
+1. (Optional) [Set the `dd-otel-span-mapping` HTTP header](#optional-map-or-filter-span-names) to map or filter spans.
+
+### Configure the exporter
+
+To send OTLP data to the Datadog OTLP intake endpoint, you need to use the OTLP HTTP Protobuf exporter. The process differs depending on whether you are using automatic or manual instrumentation for OpenTelemetry.
+
+#### Automatic instrumentation
+
+If you are using [OpenTelemetry automatic instrumentation][3], set the following environment variables, replacing `{DD_API_KEY}` with your Datadog API Key and `{YOUR_ORG]` with your organization:
+
+```shell
+export OTEL_EXPORTER_OTLP_TRACES_PROTOCOL="http/protobuf"
+export OTEL_EXPORTER_OTLP_TRACES_ENDPOINT="{{< region-param key="otlp_trace_endpoint" >}}"
+export OTEL_EXPORTER_OTLP_TRACES_HEADERS="dd-protocol=otlp,dd-api-key=${DD_API_KEY},dd-otlp-source=${YOUR_ORG}"
+```
+
+#### Manual instrumentation
+
+If you are using manual instrumentation with OpenTelemetry SDKs, configure the OTLP HTTP Protobuf exporter programmatically.
+
+<div class="alert alert-info">Replace <code>${YOUR_ENDPOINT}</code> with {{< region-param key="otlp_trace_endpoint" code="true" >}}. This is based on your <a href="/getting_started/site/">Datadog site</a>, which is {{< region-param key=dd_datacenter code="true" >}}.</div>
+
+{{< tabs >}}
+{{% tab "JavaScript" %}}
+
+The JavaScript exporter is [`exporter-trace-otlp-proto`][100]. To configure the exporter, use the following code snippet:
+
+```javascript
+const { OTLPTraceExporter } = require('@opentelemetry/exporter-trace-otlp-proto');  // OTLP http/protobuf exporter
+
+const exporter = new OTLPTraceExporter({
+  url: '${YOUR_ENDPOINT}', // Replace this with the correct endpoint
+  headers: {
+    Cdd-protocol': 'otlp', 
+    'dd-api-key': process.env.DD_API_KEY,
+    'dd-otel-span-mapping': '{span_name_as_resource_name: false}',
+    'dd-otlp-source': '${YOUR_ORG}',
+  },
+});
+```
+[100]: https://www.npmjs.com/package/@opentelemetry/exporter-trace-otlp-proto
+
+{{% /tab %}}
+
+{{% tab "Java" %}}
+
+The Java exporter is [`OtlpHttpSpanExporter`][200]. To configure the exporter, use the following code snippet:
+
+```java
+import io.opentelemetry.exporter.otlp.http.trace.OtlpHttpSpanExporter;
+
+OtlpHttpSpanExporter exporter = OtlpHttpSpanExporter.builder()
+    .setEndpoint("${YOUR_ENDPOINT}") // Replace this with the correct endpoint
+    .addHeader("dd-protocol", "otlp")
+    .addHeader("dd-api-key", System.getenv("DD_API_KEY"))
+    .addHeader("dd-otlp-source", "${YOUR_ORG}")
```

Comment:
Missing `'dd-otel-span-mapping': '{span_name_as_resource_name: true}'` in Java.
All the other languages have it.

Reply:
@julianocosta89 - Good catch! Thanks. I just pushed up some commits that address your suggestions. 🚀